mysensors/mysensors.py

```diff
@@ -141,7 +142,7 @@ def _handle_internal(self, msg):
                 self.sensors[msg.node_id].battery_level = int(msg.payload)
                 self.alert(msg.node_id)
         elif msg.sub_type == self.const.Internal.I_TIME:
-            return msg.copy(ack=0, payload=int(time.time()))
+            return msg.copy(ack=0, payload=int(calendar.timegm(time.localtime())))
```

Comment:
You don't need to convert to `int` anymore, since the return value from `calendar.timegm` is already `int`.